ArduCopter/motors.cpp

```diff
@@ -181,7 +181,19 @@ void Copter::motors_output()
     }
 
     // push all channels
-    SRV_Channels::push();
+    if (full_push) {
```

Comment:
It's a shame that the copter code needs to worry about the detail of which push() to call.  The copter code essentially knows that the SRV_Channels cannot do what we want it to.  It would be nicer if the SRV_Channel::push() call could figure that out itself somehow

Reply:
Left as-is